src/ClassFileLocator.php

```diff
@@ -122,6 +122,18 @@ public function accept()
                     if ($i > 0 && is_array($tokens[$i - 1]) && $tokens[$i - 1][0] === T_DOUBLE_COLON) {
                         break;
                     }
+
+                    // ignore anonymous classes on PHP 7.1 and greater
+                    if (PHP_VERSION_ID >= 70100
+                        && $i > 2
```

Comment:
You tell me; haven't played with tokenizing anonymous classes yet. :) What happens in the tests?